src/assets-webkit/styles/serlo-tailwind.css

```diff
@@ -255,26 +250,57 @@
   @apply absolute;
 }
 
+h1,
+h2,
+main b {
+  @apply tracking-slightestly-tighter;
+}
+body {
+  @apply tracking-slightly-tighter;
+}
+
+table.gstl_50.gssb_c {
+  @apply z-50 serlo-input-font-reset;
+  @apply mt-0.5 left-1 !right-1 !w-auto;
+  @apply sm:ml-2.5 sm:mt-0.5 w-auto;
+
+  .gssb_a td {
+    @apply !whitespace-normal;
+  }
+}
+
+/* print */
+
 @media print {
   html {
     @apply text-xs;
   }
   body {
     @apply bg-white;
   }
+  .print\:block {
+    @apply !block;
+  }
   #__next {
+    .print\:hidden,
     > header,
     > footer,
     #notification-wrapper,
     nav,
-    button,
     aside,
     h1 > span > svg {
       @apply !hidden;
     }
     main {
       @apply block w-full max-w-full relative;
     }
+    img,
+    table {
+      page-break-inside: avoid;
+    }
+    .serlo-exercise-wrapper {
+      border-bottom: 1px solid black;
+    }
   }
   .header-breadcrumbs,
   a,
```

Comment:
global styles are a bit tricky generally - you might try to add the class `print:serlo-print-style ` to the body tag (print is the media selector we have activated in the config) - this should avoid the duplication.

Reply:
a nice, I'll try that.